vault/dendron.topic.note-reference.md

```diff
@@ -97,6 +97,18 @@ This reference will reference the start of a note until it encounters the first
 
 ![[dendron.topic.note-reference.sample#^begin]]
 
+#### End Positional Reference
+
+This reference will reference the end of a note. 
+
+- NOTE: the `end` positional reference cannot be used in the `start-anchor` position of a [[#range-reference]]
+
+```md
+![[dendron.topic.note-reference.sample#^end]]
```

Comment:
Would anyone ever want to do this? Should we change the example to use a range reference instead?

Reply:
Er - ok maybe I'm misunderstanding this.  This example is showing what we _can't_ do? Why don't we instead show an example of when someone would want to use the end positional reference?

Reply:
ah, wrote the wrong example. updated this.